src/web/crate_details.rs

```diff
@@ -620,10 +636,27 @@ pub(crate) async fn get_all_platforms_inner(
     .await?
     .ok_or(AxumNope::CrateNotFound)?;
 
-    let doc_targets = MetaData::parse_doc_targets(krate.doc_targets);
+    if krate.doc_targets.is_none()
+        || krate.default_target.is_none()
+        || matched_release.target_name().is_none()
+    {
+        // FIXME: is this really OK?  just return an empty platform list in case of
```

Comment:
Maybe an error instead?

Reply:
after having thought about it longer, I think the empty list is fine

Reply:
( I'll update the comment)